.github/workflows/node.js.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
```

Comment:
I assume `$default-branch` is `main|master`?

Reply:
turns out i had to replace this value or the action didn't run. https://github.com/metaplex/arweave-cost/pull/2